### PR TITLE
Fix InvalidLaunchFileError error message.

### PR DESCRIPTION
**Pull request by hidmic**
Precisely what the title says.

### CODE REVIEW

File: launch/launch/invalid_launch_file_error.py [hunk 1]
```diff
@@ -26,11 +26,9 @@ def __init__(self, extension='', *, likely_errors=None):
...
             ', or its format is unknown'
         else:
-            self._error_message = 'Caught exception when trying to load file of format [{}]:'
-            ' {}'.format(
-                self._extension,
-                self._likely_errors[0]
-            )
+            self._error_message = (
+                'Caught exception when trying to load file of format [{}]: {}'
+            ).format(self._extension, self._likely_errors[0])
```
Comment (ivanpauno) [inline launch/launch/invalid_launch_file_error.py]: Oh, I was just formatting the second line, right? Didn't know that.

**Comment by jacobperron** [comment]
I think this resolves #321 ?

**Comment by ivanpauno** [comment]
> I think this resolves #321 ?

Was the only problem in #321 the bad message formatting? The issue is not clear to me.
If that's the case, this fixes #321.

**Comment by jacobperron** [comment]
> Was the only problem in #321 the bad message formatting? The issue is not clear to me.
If that's the case, this fixes #321.

Yes. Error messages were improved in #315, but then "regressed" when I started seeing uninformative error messages:

```
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Caught exception when trying to load file of format [{}]:
```

I've confirmed that this PR resolves my issue.

File: launch/launch/invalid_launch_file_error.py [hunk 2]
```diff
@@ -26,11 +26,9 @@ def __init__(self, extension='', *, likely_errors=None):
             self._error_message = 'The launch file may have a syntax error'
             ', or its format is unknown'
```
Comment (ivanpauno) [inline launch/launch/invalid_launch_file_error.py]: Can you correct this too?
```python3
self._error_message = \
    'The launch file may have a syntax error, or its format is unknown'
```

**Comment by hidmic** [comment]
Running CI for `launch`:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/8195/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/4171/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/6673/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/8069/)

**Pull request merged**